handbook/people-ops/travel.md

```diff
@@ -31,12 +21,30 @@ Sourcegraph generally holds two company-wide events each year, each fully covere
 - If one individual is a member of multiple teams, they will receive the same amount for each of their teams.
 - The budget can be used for online team bonding events if the team prefers.
 
+## Team travel budgets
+
+Sourcegraph is an [all-remote](../../company/remote/index.md) company. This means we all need to make an effort to travel to meet up in person and spend quality time with other team members all around the world.
```

Comment:
I think that this section (the first 3-4 paragraphs under this header) was good as an intro to this page (it seemed to be a higher-level overview of our philosophy on events, versus the other headers which are specific categories of events.). 

E.g., the last sentence in this section seems out of place now given it talks about how "there are four types of travel", but this section comes beneath two of them.

What do you think about putting this back at the top, but _just_ underneath the new COVID-19 section?

Reply:
Good point, it did look a bit funky. I just made the edit!